Assets/Scripts/Camera/CameraPosition.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class CameraPosition : MonoBehaviour
+    {
+        public Transform cameraPosition;
+
+        private void Update()
+        {
+            transform.position = cameraPosition.position;
+        }
+    }
```

Comment:
Maybe rename to `FollowObject`? And `Transform cameraPosition` to `Transform target`